### PR TITLE
Set session preset

### PR DESCRIPTION
**Pull request by manask88**

### CODE REVIEW

File: Camera.js [hunk 1]
```diff
@@ -31,12 +31,12 @@ function convertStringProps(props) {
   if (typeof props.type === 'string') {
     newProps.type = Camera.constants.Type[props.type];
   }
-  
+
```
Comment (pjcabrera) [inline Camera.js:35]: Here, you need to convert props.captureQuality from string to number.

```
if (typeof props.captureQuality === 'string') {
  newProps.captureQuality = Camera.constants.CaptureQuality[props.captureQuality];
}
```

File: ios/RCTCameraManager.m [hunk 2]
```diff
@@ -574,6 +579,11 @@ -(void)captureVideo:(NSInteger)target options:(NSDictionary *)options resolve:(R
     [self initializeCaptureSessionInput:AVMediaTypeAudio];
   }
 
+//  if ([options valueForKey:@"quality"]) {
```
Comment (pjcabrera) [inline ios/RCTCameraManager.m:604]: Why did you comment this out? This is where you fix it to take the value of the quality prop.

File: ios/RCTCameraManager.m [hunk 3]
```diff
@@ -123,7 +128,7 @@ - (id)init {
   if ((self = [super init])) {
 
     self.session = [AVCaptureSession new];
-    self.session.sessionPreset = AVCaptureSessionPresetHigh;
+    self.session.sessionPreset = AVCaptureSessionPresetHigh;
```
Comment (pjcabrera) [inline ios/RCTCameraManager.m:144]: This is what we want to get rid of. Why provide a default value in defaultProps on the JS side if you're leaving it hard coded here?

Comment (manask88) [inline ios/RCTCameraManager.m:144]: thanks for the comments @pjcabrera . I understand this things don't make much sense - this is still WIP and I was just making sure my code was not the one breaking the project

File: ios/RCTCameraManager.m (hunk 2 above)
Comment (manask88) [inline ios/RCTCameraManager.m:604]: thanks for the comments @pjcabrera . I understand this things don't make much sense - this is still WIP and I was just making sure my code was not the one breaking the project. However, your comment is helpful since it lets me know that im on the right track!

File: ios/RCTCameraManager.m (hunk 3 above)
Comment (pjcabrera) [inline ios/RCTCameraManager.m:144]: Sorry, didn't realize it was WIP

File: Camera.js [hunk 4]
```diff
@@ -47,7 +47,7 @@ export default class Camera extends Component {
     FlashMode: CameraManager.FlashMode,
     TorchMode: CameraManager.TorchMode
   };
-  
+
```
Comment (pjcabrera) [inline Camera.js:51]: While I'm bugging you, revert these changes to the whitespace before you submit the PR to his repo. I believe you didn't do this, maybe Atom changed them somehow. But these extra characters are his coding style preference, and changing them just adds noise to the PR.

File: Camera.js [hunk 5]
```diff
@@ -83,25 +87,26 @@ export default class Camera extends Component {
       PropTypes.number
     ])
   };
-  
+
```
Comment (pjcabrera) [inline Camera.js:93]: revert these changes to the whitespace before you submit the PR to his repo. I'm only pointing out a few of these, you can find the rest by looking at your PR on SourceTree or other Git tool.
